app/components/Send/SendPanel/SendRecipientList/SendRecipientListItem/index.jsx

```diff
@@ -143,7 +143,7 @@ class SendRecipientListItem extends Component<Props> {
 
     return (
       <li className={styles.sendRecipientListItem}>
-        <div className={styles.rowNumber}>{`0${index + 1}`}</div>
+        <div className={styles.rowNumber}>{`${`0${index + 1}`.slice(-2)}`}</div>
```

Comment:
this feels weird with .slice(-2) but not sure what is happening. if everything is working correctly it should be fine